### PR TITLE
Add git hash sending to larabug and allow filtering of sensitive info…

### PR DESCRIPTION
**Pull request by xHeinrich**
Adds the ability to blacklist variables from sending to the server, also adds a git release hash to send to the server

### CODE REVIEW

File: config/larabug.php [hunk 1]
```diff
@@ -43,4 +43,16 @@
         'Symfony\Component\HttpKernel\Exception\NotFoundHttpException',
     ],
 
+    /*
+     * Filter out these variables before sending them to larabug
+     */
+    'blacklist' => [
+        'password',
+        'authorization'
```
Comment (Cannonb4ll) [inline config/larabug.php]: What happens if there's a key Authorization? Does the array_walk function strtolower them or? Let me know 👍

Comment (xHeinrich) [reply config/larabug.php]: True, I'll have to have a think on how to solve that

Comment (Cannonb4ll) [reply config/larabug.php]: As far as I am concerned you can just strotolower all values (from config and from the incoming data) and compare these, so you're sure those keys will not be logged.

Comment (xHeinrich) [reply config/larabug.php]: Updated to remove the keys no matter the casing and added a test. I'm not a fan of lowercasing the keys in the constructor but not really sure of a better way to memoize them.

**Comment by Cannonb4ll** [comment]
💙👏

**Pull request merged**